Compentio.SourceMapper.Generator/Processors/DependencyInjection/DependencyInjection.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace Compentio.SourceMapper.Processors.DependencyInjection
+﻿using Compentio.SourceMapper.Resources;
+
+namespace Compentio.SourceMapper.Processors.DependencyInjection
 {
     internal class DependencyInjection
     {
-        internal DependencyInjectionType DependencyInjectionType { get; set; }
-        internal string DependencyInjectionClassName { get; }  = "MappersDependencyInjectionExtensions";
+        internal virtual DependencyInjectionType DependencyInjectionType { get; set; }
+        internal string DependencyInjectionClassName { get; }  = AppStrings.DependencyInjectionClassName;
```

Comment:
It looks like a good idea to have Class names in resources!

Reply:
Is it irony? :) 
I made that as I mentioned above.

Reply:
No, here is, class name (because it is a part of generated code) can be changed and resx good place for it.